aldryn_faq/models.py

```diff
@@ -24,6 +24,7 @@
 
 from .cms_appconfig import FaqConfig
 from .managers import CategoryManager, RelatedManager
+from .utils import namespace_is_apphooked
```

Comment:
I think a better name for this utility is ``is_valid_namespace``.

And considering that everytime it's called like namespace_is_apphooked(getattr(app_config, 'namespace', None))
I think would be good to create a second utility called is_valid_app_config which simply does this ``getattr(app_config, 'namespace', None)`` and passes the namespace to ``is_valid_namespace``.